Jenkinsfile

```diff
@@ -63,6 +63,11 @@ pipeline {
         echo "Run Commmands to trigger build"
       }
     }
+    stage("Code Coverage of JavaScript files") {
```

Comment:
I guess @arpit-agarwal is mentioned about this stage. Please confirm

Reply:
At this stage sh "tox -e javascript" shell script is added.